src/py/flwr/cli/new/templates/app/pyproject.hf.toml.tpl

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "$package_name"
+version = "1.0.0"
+description = ""
+authors = [
+    { name = "The Flower Authors", email = "hello@flower.ai" },
+]
+license = {text = "Apache License (2.0)"}
+dependencies = [
+    "flwr[simulation]>=1.8.0,<2.0",
```

Comment:
Let's fix this (and all of the other examples) to `>=1.9.0,<2.0.0`

Reply:
@danieljanes Can we move this to another PR? Same with the comment below, as they apply to all templates

Reply:
yes